### ISSUE TITLE
Add OpenHatch's "I want to help" button

### ISSUE THREAD

**Issue by arsturges** [arsturges/wikidiscography#7]
To get more visibility and open the site for greater access to those who want to learn how to contribute, follow the instructions at this web site to add a button that says "I want to help":

http://openhatch.org/blog/2010/let-people-help-your-project-give-them-a-button-to-click/

The best place to add this button is probably to the "/pages/contributing" page.

**Comment by cczona** [comment]
That link 404s.

**Comment by arsturges** [comment]
Good catch. Don't know why it's doing that. At this gist is the code that OpenHatch generated for this project's button.
--Andrew
git://gist.github.com/1244278.git
